webview/src/plots/components/ZoomedInPlot.tsx

```diff
@@ -66,6 +57,7 @@ export const ZoomedInPlot: React.FC<ZoomedInPlotProps> = ({
           export: true,
           source: false
         }}
+        onNewView={onNewView}
```

Comment:
Really surprised that I forgot about this prop when working on this earlier 😅. Looks to be working without issues!